Weather/Scenes/TimedForecast/TimedForecastCellView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  TimedForecastCellView.swift
+//  Weather
+//
+//  Created by Vladislav Kondrashkov on 10/22/18.
+//  Copyright © 2018 Vladislav Kondrashkov. All rights reserved.
+//
+
+import UIKit
+
+final class TimedForecastCellView: UICollectionViewCell {
```

Comment:
Usually cell views are called "SomethingCollectionViewCell" in this case it would be TimedForecastCollectionViewCell